classes/PodsAPI.php

```diff
@@ -1678,17 +1678,35 @@ public function save_pod( $params, $sanitized = false, $db = true ) {
 			unset( $params->create_extend );
 		}
 
-		$load_params = array();
+		if ( isset( $params->pod ) && $params->pod instanceof \Pods\Whatsit\Pod ) {
+			$pod = $params->pod;
 
-		if ( ! empty( $params->id ) ) {
-			$load_params['id'] = $params->id;
-		} elseif ( ! empty( $params->old_name ) ) {
-			$load_params['name'] = $params->old_name;
-		} elseif ( ! empty( $params->name ) ) {
-			$load_params['name'] = $params->name;
+			unset( $params->pod );
+		} else {
+			$load_params = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: f503b56c964c255570fd49e58c913312</sub>